src/routes/news/containers/TunewsPage.js

```diff
@@ -48,6 +54,17 @@ export class TunewsPage extends React.PureComponent<PropsType> {
 
   render () {
     const { tunews, language, city, t, fetchMoreTunews, hasMore, isFetchingFirstTime, isFetching, cities } = this.props
+
+    if (this.props.areCitiesFetching) {
```

Comment:
AFAICS this is missing in the props. Is this needed?

Reply:
Fixed! Yes it's needed or there will be an error because the cities are null while fetching